Makefile

```diff
@@ -154,6 +155,10 @@ test-python:
 lint:
 	$(RUN) linkml-lint $(SOURCE_SCHEMA_PATH)
 
+gen-artefacts: $(PYMODEL) $(JSONSCHEMA_DIR)
+	$(RUN) gen-json-schema $(SOURCE_SCHEMA_PATH) > $(JSONSCHEMA_DIR)/bertron_schema.json
+	$(RUN) gen-pydantic $(SOURCE_SCHEMA_PATH) > $(PYMODEL)/bertron_schema_pydantic.py
+
```

Comment:
shortcut for generating the jsonschema and pydantic versions of the schema after updating the linkml version